pkg/resourcesets/collector.go

```diff
@@ -436,17 +436,21 @@ func writeToBackup(ctx context.Context, resource map[string]interface{}, backupP
 	if err != nil {
 		return fmt.Errorf("error converting resource to JSON: %v", err)
 	}
-	if transformer != nil && !util.IsDefaultEncryptionTransformer(transformer) {
-		encrypted, err := transformer.TransformToStorage(ctx, resourceBytes, value.DefaultContext(additionalAuthenticatedData))
-		if err != nil {
-			return fmt.Errorf("error converting resource to JSON: %v", err)
-		}
 
-		resourceBytes, err = json.Marshal(encrypted)
+	// Since k8s 1.32 we cannot verify a transformer must be run, so it's always run now.
+	maybeEncrypted, err := transformer.TransformToStorage(ctx, resourceBytes, value.DefaultContext(additionalAuthenticatedData))
```

Comment:
I think that because of those factors, this is a new change in dataflow/processing that non-encrypted restores used to skip, but will now run all the time. So ultimately this likely doesn't change too much at the end of the day, but is technically a new area that an error could come from.